gromacs/fileformats/top.py

```diff
@@ -0,0 +1,1243 @@
+
+#
+# JD: adopted from 'pytopol' from https://github.com/resal81/PyTopol
```

Comment:
Need module doc string and acknowledgement of https://github.com/resal81/PyTopol